rules/kotlin/kotlin.bzl

```diff
@@ -1,5 +1,46 @@
 load("@io_bazel_rules_kotlin//kotlin:jvm.bzl", _kt_jvm_library = "kt_jvm_library")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", _kt_compiler_plugin = "kt_compiler_plugin")
+load("@grab_bazel_common//rules/android/lint:defs.bzl", "lint", "lint_sources")
+load("@grab_bazel_common//rules/android/lint:providers.bzl", "LINT_ENABLED")
 
 kt_jvm_library = _kt_jvm_library
 kt_compiler_plugin = _kt_compiler_plugin
+
+def kt_jvm_library_interal(
+        name,
+        exec_properties = None,
+        lint_options = {},
+        **attrs):
+    lint_sources_target = "_" + name + "_lint_sources"
+    lint_baseline = lint_options.get("baseline", None)
+
+    lint_sources(
```

Comment:
Can we add an `if` check on `lint_options.enabled` to add these `lint_sources` and `lint` only when it is true? 

This way we don't by default enable android lint on kotlin targets. Later we can generate this option from grazel.

Reply:
done